### DISCUSSION TITLE
Is it possible to add a sleep mode setting?

### DISCUSSION THREAD

**Discussion by giceli** [navidrome/navidrome#3570] [Ideas]
i install navidrome in iStoreOS with docker,
My Hard Disk Drive will go into sleep mode if there is no read or write activity for 10 minutes
but when i running navidrome, i find that my My Hard Disk Drive is always working
Is there a way to put the service into sleep mode when it finds that I haven't listened to music for a long time,
and then automatically start the service next time I continue to listen to music?

**Comment by kgarner7** [comment]
If you haven't changed it, the default scan schedule is 1 minute (https://www.navidrome.org/docs/usage/configuration-options/#:~:text=false-,scanschedule,-ND_SCANSCHEDULE), so that's _likely_ what's causing this behavior. Consider adjusting this value as needed.

**Reply by giceli** [reply]
it works， thanks
